style.css

```
@@ -1,4 +1,53 @@
 * {
     margin: 0;
     box-sizing: border-box;
+}
+
+.navegation {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    height: 10rem;
+}
+
+.logo-wrap img {
+    width: 97px;
+    height: 59px;
+    flex-shrink: 0;
+}
+
+.navegation img {
```

é bom evitar adicionar estilos a tags html diretamente, se vc precisar é melhor adicionar uma classe no html e usar ela no css.

E tem um pequeno erro de digitação.
```suggestion
.navigation img {
```